src/content/docs/faq.mdx

```diff
@@ -475,6 +477,23 @@ If you'd like to segment availability of Shorebird patches, there are 4 potentia
 
 Yes. Apps built with Shorebird will continue to function normally (as if they had been built without Shorebird), including those that have had patches installed.
 
+### Now that I'm using Shorebird, do I still need to send my app to the stores?
```

Comment:
Should this be "will shorebird handle all distribution for me"? Not all of our users distribute through the stores.

Reply:
We actually already have that question: https://docs.shorebird.dev/faq/#does-shorebird-submit-to-the-stores-for-me. I can link to one from the other?

Reply:
I've re-sorted the answers.